apps/rush-lib/src/logic/PackageJsonUpdater.ts

```diff
@@ -268,13 +269,15 @@ export class PackageJsonUpdater {
    * Selects an appropriate version number for a particular package, given an optional initial SemVer spec.
    * If ensureConsistentVersions, tries to pick a version that will be consistent.
    * Otherwise, will choose the latest semver matching the initialSpec and append the proper range style.
+   * @param projects - the project which will have its package.json updated
    * @param packageName - the name of the package to be used
    * @param initialSpec - a semver pattern that should be used to find the latest version matching the spec
    * @param implicitlyPinnedVersion - the implicitly preferred (aka common/primary) version of the package in use
    * @param rangeStyle - if this version is selected by querying registry, then this range specifier is prepended to
    *   the selected version.
    */
   private async _getNormalizedVersionSpec(
+    projects: RushConfigurationProject[],
```

Comment:
Do we support multiple projects?

Reply:
I don't think so.. That's why I'm curious about why it has an array here. The statement is like this: projects = [currentProject]; Don't you think we actually don't need this array here?

Reply:
Looks like this is used when the `--all` flag is specified.